### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by nd0ut**
Updates:
 - update widget version to `3.x`
 - update plugin version to `2.2.0`
 - pass `integration` setting to the widget

Also I pushed `v2-dev-env` branch with some development environment.

### CODE REVIEW

**Comment by nd0ut** [comment]
[Widget & CKEditor: обновить](https://trello.com/c/alYu5WGU/216-widget-ckeditor-%D0%BE%D0%B1%D0%BD%D0%BE%D0%B2%D0%B8%D1%82%D1%8C)

File: plugin.js [hunk 1]
```diff
@@ -19,6 +19,17 @@ CKEDITOR.plugins.add('uploadcare', {
     if ( ! ('crop' in config)) {
       config.crop = '';
     }
+    
+    applyIntegrationSetting();
+
+    function applyIntegrationSetting() {
+      var editorVersion = CKEDITOR.version;
+      var pluginVerion = '2.1.2';
```
Comment (dmitry-mukhin) [inline plugin.js]: not anymore :p

File: plugin.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Uploadcare CKeditor plugin
-// Version: 2.1.2
+// Version: 2.2.0
```
Comment (Zmoki) [inline plugin.js]: version - `3.0.0`

File: plugin.js [hunk 3]
```diff
@@ -19,6 +19,17 @@ CKEDITOR.plugins.add('uploadcare', {
     if ( ! ('crop' in config)) {
       config.crop = '';
     }
+    
+    applyIntegrationSetting();
+
+    function applyIntegrationSetting() {
+      var editorVersion = CKEDITOR.version;
+      var pluginVerion = '2.2.0';
```
Comment (Zmoki) [inline plugin.js]: move version to top of the file

**Pull request closed without merging**